src/containers/Node/Node.tsx

```diff
@@ -0,0 +1,187 @@
+import * as React from 'react';
+import {useRouteMatch} from 'react-router';
+import cn from 'bem-cn-lite';
+import {useDispatch, useSelector} from 'react-redux';
+import _ from 'lodash';
+
+import {Loader, Tabs} from '@yandex-cloud/uikit';
+import {Link} from 'react-router-dom';
+
+import {TABLETS, STORAGE, NODE_PAGES, OVERVIEW, STRUCTURE} from './NodePages';
+import Tablets from '../Tablets/Tablets';
+import Storage from '../Storage/Storage';
+import NodeOverview from './NodeOverview/NodeOverview';
+import NodeStructure from './NodeStructure/NodeStructure';
+
+import {getNodeInfo, resetNode} from '../../store/reducers/node';
+import routes, {CLUSTER_PAGES, createHref} from '../../routes';
+import {setHeader} from '../../store/reducers/header';
+import {AutoFetcher} from '../../utils/autofetcher';
+
+import './Node.scss';
+
+const b = cn('node');
+
+export const STORAGE_ROLE = 'Storage';
+
+const headerNodes = {
+    text: CLUSTER_PAGES.nodes.title,
+    link: createHref(routes.cluster, {activeTab: CLUSTER_PAGES.nodes.id}),
+};
+
+const autofetcher = new AutoFetcher();
+
+interface NodeProps {
+    additionalNodesInfo?: any;
+    className?: string;
+}
+
+function Node(props: NodeProps) {
+    const dispatch = useDispatch();
+
+    const {wasLoaded, loading, error} = useSelector((state: any) => state.node);
+
+    const {node, nodeHost} = useSelector((state: any) => {
+        const data = state.node.data;
+        const node = data.SystemStateInfo ? data.SystemStateInfo[0] : undefined;
+        return {node, nodeHost: node?.Host};
```

Comment:
Do you know the main difference between comparisons in mapStateToProps and useSelector? The way you structure useSelector's result tells me that https://react-redux.js.org/api/hooks#equality-comparisons-and-updates may have some insights regarding improving the performance ;).

Reply:
Brilliant! I fixed some issues and repaired scroll again :)